Update installation_guide.rst (#3073)
rotki/rotki: docs/installation_guide.rst

should be able to access rotki frontend, open your browser and go to
it at ``http://localhost:8084``. You should be able to see the rotki login screen.

> Time in Profit / Loss Report is in wrong.

Reviewer: This change doesn't affect only to the Profit and Loss Report but to every action. I believe a better title is

```suggestion
Modify the timezone used by rotki
```

Author: The UI is running in the correct time zone also with no timezone set in the container.
But your title should also be fine.

---------------------------

> To fix this problem, it is necessary to set the TimeZone enviroment variable with the command ``-e TZ=Timezone`` when starting the container::

Reviewer: ```suggestion
To set the timezone in the docker environment you can use the option ``-e TZ=Timezone`` when starting the container:
```
